### PR TITLE
44019 content diff for minkotsversion

### PR DESCRIPTION
**Pull request by paigecalvert**
SC: https://app.shortcut.com/replicated/story/44019/content-diff-remove-minkotsversion-limitations

https://deploy-preview-182--replicated-docs.netlify.app/reference/custom-resource-application#minkotsversion-beta

### CODE REVIEW

File: docs/reference/custom-resource-application.md [hunk 1]
```diff
@@ -182,25 +182,24 @@ For more information about the KOTS add-on, see [KOTS add-on](https://kurl.sh/do
...
+:::
 
-`minKotsVersion` is not supported in these cases because when the minimum version increases, the admin console can error when retrieving it. For more information about these error messages, see [Guidance for informing users of the need to update](#guidance-for-informing-users-of-the-need-to-update) below.
+### How the Admin Console Handles minKotsVersion
 
-If the minimum KOTS version then decreases in the next release, KOTS successfully retrieves that release, which means the intermediate release is lost.
+When you promote a new release that specifies a minimum KOTS version later than what a user currently has deployed, that application version appears in the version history of the admin console after the user checks for updates. However, it is not downloaded.
 
-### Guidance for informing users of the need to update
+The admin console temporarily displays an error message that informs the user that they must update KOTS in order to download the application version. This error is also displayed when the user checks for updates with the [`kots upstream upgrade`](/kots-cli/upstream/upgrade/) command.
```
Comment (jonquil2002) [inline docs/reference/custom-resource-application.md:197]: Suggest changing verb tense, perhaps in a later rewrite, to:

"This error displays when..."

**Review by jonquil2002 — APPROVED** [review]
lgtm. One minor suggestion for verb tense that can wait til a future rewrite.

**Comment by paigecalvert** [comment]
Great suggestion @jonquil2002 ; just committed the change

**Review by jonquil2002 — APPROVED** [review]
lgtm

**Pull request merged**